rocAL/rocAL_pybind/example/tf_detection.py

```diff
@@ -0,0 +1,232 @@
+        'image/object/bbox/xmax': 'image/object/bbox/xmax',
+        'image/object/bbox/ymax': 'image/object/bbox/ymax',
+        'image/filename': 'image/filename'
+    }
+    num_threads = 1
+    device_id = 0
+    numClasses = 91     
+    
+    random_seed = random.SystemRandom().randint(0, 2**32 - 1)
+    crop=300
+
+    local_rank = 0
+    world_size = 1
+    rocal_cpu= True
+    rocal_device = 'cpu' if rocal_cpu else 'gpu'
+    decoder_device = 'cpu' if rocal_cpu else 'mixed'
+    device_memory_padding = 211025920 if decoder_device == 'mixed' else 0
+    host_memory_padding = 140544512 if decoder_device == 'mixed' else 0
+    
+    pipe = Pipeline(batch_size=batch_size, num_threads=num_threads,device_id=device_id, seed=2, rocal_cpu=_rocal_cpu, tensor_dtype=types.FLOAT)
+    print(pipe)
+    with pipe:
+        inputs = fn.readers.tfrecord(path=data_path, index_path = "", reader_type=TFRecordReaderType, user_feature_key_map=featureKeyMap,random_shuffle=True,
+            features={
+                    'image/encoded': tf.io.FixedLenFeature((), tf.string, ""),
+                    'image/class/label': tf.io.FixedLenFeature([1], tf.int64,  -1),
+                    'image/class/text': tf.io.FixedLenFeature([], tf.string, ''),
+                    'image/object/bbox/xmin': tf.io.VarLenFeature(dtype=tf.float32),
+                    'image/object/bbox/ymin': tf.io.VarLenFeature(dtype=tf.float32),
+                    'image/object/bbox/xmax': tf.io.VarLenFeature(dtype=tf.float32),
+                    'image/object/bbox/ymax': tf.io.VarLenFeature(dtype=tf.float32),
+                    'image/filename': tf.io.FixedLenFeature((), tf.string, "")
+                    }
+        )
+        jpegs = inputs["image/encoded"]
+        images = fn.decoders.image_random_crop(jpegs,user_feature_key_map=featureKeyMap, output_type=types.RGB,
+                                                      random_aspect_ratio=[0.8, 1.25],
+                                                      random_area=[0.1, 1.0],
+                                                      num_attempts=100,path = data_path)
+                                                      
+        
+        resized = fn.resize(images, resize_width=400, resize_height=400,rocal_tensor_output_type = types.UINT8, rocal_tensor_layout = types.NHWC)
+        cmnp = fn.crop_mirror_normalize(resized, device="cpu",
+                                            rocal_tensor_layout = types.NHWC,
+                                            rocal_tensor_output_type = types.FLOAT,
+                                            output_dtype = types.FLOAT,
+                                            crop=[300, 300],
+                                            mirror=0,
+                                            image_type=types.RGB,
+                                            mean=[0,0,0],
+                                            std=[1,1,1],crop_d=3)
```

Comment:
we dont use the crop_d here. Please remove this. Refer the classification example for CMN params.
Use a certain mean & std dev, mirror values & test the outputs too